src/transfer.ts

```diff
@@ -114,6 +143,7 @@ const transferInStatusPreTxResponseSchema = z.object(
     fiatAccountId: fiatAccountIdSchema,
     transferId: z.string(),
     transferAddress: z.string(),
+    userActionDetails: userActionDetailsSchema.optional(),
```

Comment:
from rereading the spec update where this field was added, it's not totally clear to me if the field is required for transfers that originally required a user action, but the transfer has reached a state where it no longer requires the action. Given that you included it here but non in the post-transaction response schema, it seems like the intention was to only include it if the user action was still required. might be good to clarify that at some point.

Reply:
That's actually not the intent with the `optional()` here. It's optional simply because certain transfers in don't _ever_ require user action at any point (e.g. if a provider is able to debit a user's fiat account on their behalf).